Resources/Prototypes/_Goobstation/Entities/Mobs/Cyborgs/borg_chassis.yml

```
@@ -44,4 +44,71 @@
+    interactSuccessSound:
+      path: /Audio/Voice/Human/malescream_2.ogg
+
+- type: entity
+  id: PlayerBorgUrist
+  parent: BorgChassisUrist
+  suffix: Battery, Module
+  components:
+  - type: ContainerFill
+    containers:
+      borg_brain:
+        - PositronicBrain
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)